dotcom-rendering/src/web/components/LiveBlogEpic.importable.tsx

```diff
@@ -249,12 +250,17 @@ const Fetch = ({
 	}
 	log('dotcom', 'LiveBlogEpic has a module');
 
+	const props = {
```

Comment:
```suggestion
	// We add submitComponentEvent to the props like this here on the client because
	//  the ophan record function is not available on the server
	const props = {
```